tools/nni_cmd/updater.py

```diff
@@ -80,32 +80,57 @@ def update_searchspace(args):
     args.port = get_experiment_port(args)
     if args.port is not None:
         if update_experiment_profile(args, 'searchSpace', content):
-            print('INFO: update %s success!' % 'searchSpace')
+            print_normal('Update %s success!' % 'searchSpace')
         else:
-            print('ERROR: update %s failed!' % 'searchSpace')
+            print_error('Update %s failed!' % 'searchSpace')
+
 
 def update_concurrency(args):
     validate_digit(args.value, 1, 1000)
     args.port = get_experiment_port(args)
     if args.port is not None:
         if update_experiment_profile(args, 'trialConcurrency', int(args.value)):
-            print('INFO: update %s success!' % 'concurrency')
+            print_normal('Update %s success!' % 'concurrency')
         else:
-            print('ERROR: update %s failed!' % 'concurrency')
+            print_error('Update %s failed!' % 'concurrency')
 
 def update_duration(args):
     #parse time, change time unit to seconds 
     args.value = parse_time(args.value)
     args.port = get_experiment_port(args)
     if args.port is not None:
         if update_experiment_profile(args, 'maxExecDuration', int(args.value)):
-            print('INFO: update %s success!' % 'duration')
+            print_normal('Update %s success!' % 'duration')
         else:
-            print('ERROR: update %s failed!' % 'duration')
+            print_error('Update %s failed!' % 'duration')
 
 def update_trialnum(args):
     validate_digit(args.value, 1, 999999999)
     if update_experiment_profile(args, 'maxTrialNum', int(args.value)):
-        print('INFO: update %s success!' % 'trialnum')
+        print_normal('Update %s success!' % 'trialnum')
+    else:
+        print_error('Update %s failed!' % 'trialnum')
+
+def import_data(args):
+    '''import additional data to the experiment'''
```

Comment:
suggest to have a list of supported tuners

Reply:
Done